src/huggingface_hub/hf_api.py

```diff
@@ -1431,7 +1428,7 @@ def list_repo_files(
         )
         return [f.rfilename for f in repo_info.siblings]
 
-    @_deprecate_positional_args
+    @_deprecate_positional_args(version="0.8")
```

Comment:
Is this change necessary?

Reply:
I just made the `version` keyword required to avoid having a deprecation for which we don't know at which version we want to remove it. Also related to this https://github.com/LysandreJik/huggingface_hub/pull/8#discussion_r938029960 I think we should increase to version 0.12 to deprecate all positional argument at the same time. It doesn't seem we are in a hurry.